src/forms/NumberInput.jsx

```diff
@@ -16,16 +16,15 @@ export const INCREMENT_BTN_CLASS = 'incrementButton';
 class NumberInput extends React.Component {
 	constructor(props) {
 		super(props);
-		this.state = {
-			value: props.value || '',
```

Comment:
This was setting value to '' whenever `props.value` was 0 because of how javascript treats falsey values.